node/config/types.go

```diff
@@ -53,17 +50,15 @@ type Logging struct {
 type StorageMiner struct {
 	Common
 
-	Subsystems    MinerSubsystemConfig
-	Dealmaking    DealmakingConfig
-	IndexProvider IndexProviderConfig
-	Proving       ProvingConfig
-	Sealing       SealingConfig
-	Storage       SealerConfig
-	Fees          MinerFeeConfig
-	Addresses     MinerAddressConfig
-	DAGStore      DAGStoreConfig
-
-	HarmonyDB HarmonyDB
+	Subsystems   MinerSubsystemConfig
+	Dealmaking   DealmakingConfig
+	Proving      ProvingConfig
+	Sealing      SealingConfig
+	Storage      SealerConfig
+	Fees         MinerFeeConfig
+	Addresses    MinerAddressConfig
+	HarmonyDB    HarmonyDB
+	EnableLibp2p bool
```

Comment:
do we even need this anymore? can we just force a `false` when we call `ConfigCommon()` because it's not used anywhere else in miner is it?

Reply:
@rvagg We need it in the itests. https://github.com/filecoin-project/lotus/blob/42743e73c074909d9f490dacaf3e5e99224e7301/itests/kit/ensemble.go#L937

Reply:
> connects one full node to the provided full nodes

not the miners though, only full nodes

Reply:
Yeah fixed this.